src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/PersonSearchRequest.kt

```diff
@@ -24,7 +22,7 @@ data class PersonSearchRequest(
       return PersonSearchRequest(
         pncNumber = person.otherIdentifiers?.pncIdentifier?.pncId,
         crn = person.otherIdentifiers?.crn,
-        forenameOne = person.givenName,
+        firstName = person.givenName,
```

Comment:
It's my impression that we don't need or use a lot of this functionality. However might it be worth adding in person.middleName here?